lib/scientist/experiment.rb

```diff
@@ -87,6 +87,16 @@ def before_run(&block)
     @_scientist_before_run = block
   end
 
+  # Define a block of code to run after an experiment completes, if the experiment
+  # is enabled.
+  #
+  # The block takes no arguments.
```

Comment:
The block takes the result of the experiment as the argument, right?

Reply:
It does, thanks for catching that! Fixed in 6ea1efc6f0cfc6393123546273b0cdef0eba2f53.